react/Tabs/index.jsx

```diff
@@ -1,66 +1,39 @@
-import React, { Component } from 'react'
+import React from 'react'
 import styles from './styles.styl'
 import classnames from 'classnames'
 
-export const Tab = function ({ name, children, className, active, activeClass, changeTab }) {
-  const activeStyle = {
-    [styles['coz-tab--active']]: active
-  }
-  if (activeClass) {
-    activeStyle[activeClass] = active
-  }
-  return <div
-    className={classnames(
-      styles['coz-tab'],
-      className,
-      activeStyle)}
-    onClick={() => changeTab(name)}>{
-    children
-  }</div>
-}
-
-export const TabList = function ({ children, activeTab, changeTab, className }) {
-  return <div className={classnames(styles['coz-tab-list'], className)}>{
-    React.Children.map(children, child =>
-      React.cloneElement(child, {
-        active: child.props.name === activeTab,
-        changeTab
-      }))
-  }</div>
-}
+export const Tab = ({ children, className }) => (
+  <div className={classnames(styles['coz-tab-panel'], className)}>{children}</div>
+)
 
-export const TabPanel = function ({ children, activeTab, name, changeTab, className }) {
-  return activeTab === name ? <div className={classnames(styles['coz-tab-panel'], className)}>{
-    children
-  }</div> : null
-}
-
-export const TabPanels = function ({ children, activeTab, name, changeTab, className }) {
-  const extra = { activeTab, changeTab }
-  return <div className={classnames(styles['coz-tab-panels'], className)}>{
-    React.Children.map(children, child => React.cloneElement(child, extra))
-  }</div>
-}
-
-export class Tabs extends Component {
+export class Tabs extends React.Component {
   constructor (props) {
     super(props)
-    this.changeTab = this.changeTab.bind(this)
-  }
-  getInitialState () {
-    return { activeTab: this.props.initialActiveTab }
+    this.state = {
+      active: this.props.initialActiveTab
+    }
   }
 
-  changeTab (tabName) {
-    this.setState({ activeTab: tabName })
+  isActiveTab (tabName) {
+    return this.state.active === tabName
   }
 
-  render ({ children, className }, { activeTab }) {
-    const changeTab = this.changeTab
-    return <div className={classnames(styles['coz-tabs'], className)}>{
-      React.Children.map(children, child =>
-        React.cloneElement(child, { activeTab, changeTab })
-      )
-    }</div>
+  render () {
+    return (
+      <div className={classnames(styles['coz-tabs'], this.props.className)}>
+        <div className={classnames(styles['coz-tab-list'])}>
+          {React.Children.map(this.props.children, tab => (
+            <div
```

Comment:
SHouldn’t the tabs be buttons ? See https://www.deque.com/blog/a11y-support-series-part-1-aria-tab-panel-accessibility/. You are also missing an `aria-selected` attribute here.